[2024-07-24T05:37:00.108+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T05:37:00.277+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [queued]>
[2024-07-24T05:37:00.319+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [queued]>
[2024-07-24T05:37:00.320+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-24T05:37:00.413+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): database_connection> on 2024-07-24 00:40:00+00:00
[2024-07-24T05:37:00.478+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=66) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T05:37:00.480+0000] {standard_task_runner.py:64} INFO - Started process 99 to run task
[2024-07-24T05:37:00.538+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_Fraud', 'database_connection', 'scheduled__2024-07-24T00:40:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/server_DAG.py', '--cfg-path', '/tmp/tmpfjw5om7t']
[2024-07-24T05:37:00.540+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask database_connection
[2024-07-24T05:37:01.571+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [running]> on host fc934038a502
[2024-07-24T05:37:02.144+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='fraud_detector' AIRFLOW_CTX_DAG_ID='ETL_Fraud' AIRFLOW_CTX_TASK_ID='database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T00:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-24T00:40:00+00:00'
[2024-07-24T05:37:02.149+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T05:37:02.150+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T05:37:02.151+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "database_connection"']
[2024-07-24T05:37:02.262+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T05:37:02.270+0000] {subprocess.py:93} INFO - database_connection
[2024-07-24T05:37:02.272+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T05:37:02.273+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T05:37:02.409+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_Fraud, task_id=database_connection, run_id=scheduled__2024-07-24T00:40:00+00:00, execution_date=20240724T004000, start_date=20240724T053700, end_date=20240724T053702
[2024-07-24T05:37:02.521+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T05:37:02.607+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T05:37:02.612+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-24T09:51:53.707+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T09:51:53.900+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [queued]>
[2024-07-24T09:51:53.970+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [queued]>
[2024-07-24T09:51:53.971+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-24T09:51:54.067+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): database_connection> on 2024-07-24 00:40:00+00:00
[2024-07-24T09:51:54.093+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T09:51:54.095+0000] {standard_task_runner.py:64} INFO - Started process 119 to run task
[2024-07-24T09:51:54.235+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_Fraud', 'database_connection', 'scheduled__2024-07-24T00:40:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/server_DAG.py', '--cfg-path', '/tmp/tmp0rdoed7k']
[2024-07-24T09:51:54.252+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask database_connection
[2024-07-24T09:51:54.412+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_Fraud.database_connection scheduled__2024-07-24T00:40:00+00:00 [running]> on host 0542575fe516
[2024-07-24T09:51:55.016+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='fraud_detector' AIRFLOW_CTX_DAG_ID='ETL_Fraud' AIRFLOW_CTX_TASK_ID='database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T00:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-24T00:40:00+00:00'
[2024-07-24T09:51:55.018+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T09:51:55.019+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T09:51:55.020+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "database_connection"']
[2024-07-24T09:51:55.161+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T09:51:55.174+0000] {subprocess.py:93} INFO - database_connection
[2024-07-24T09:51:55.175+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T09:51:55.175+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T09:51:55.228+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_Fraud, task_id=database_connection, run_id=scheduled__2024-07-24T00:40:00+00:00, execution_date=20240724T004000, start_date=20240724T095153, end_date=20240724T095155
[2024-07-24T09:51:55.385+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T09:51:55.624+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T09:51:55.633+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
